src/globus_sdk/login_flows/command_line_login_flow_manager.py

```diff
@@ -13,6 +15,13 @@
     from globus_sdk.globus_app import GlobusAppConfig
 
 
+class CommandLineLoginFlowEOFError(GlobusError):
```

Comment:
```suggestion
class CommandLineLoginFlowEOFError(GlobusError, EOFError):
```

Best to keep this as catchable with an `except EOFError` even if we want custom typing/messaging.

Reply:
Just checked to confirm, and the inheritance is
```
BaseException
 └── Exception
      └── EOFError
```
which is key to my agreement here. I don't think we're contaminating our error class here with an unrelated inheritance structure if we do this, so IMO it's a good idea.